### PR TITLE
DDF-3875 Add admin-federation webapp

### PR DESCRIPTION
**Pull request by alexaabrd**
#### What does this PR do?
creates an endpoint with a single tabbed view for all federation configurations

#### Who is reviewing it? 
@SmithJosh @vinamartin @djblue 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams/ui
#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler 
@brendan-hofmann
@clockard
#### What are the relevant tickets?
[DDF-3875](https://codice.atlassian.net/browse/DDF-3875)
#### Screenshots (if appropriate)
image

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
❓ Could we use ace instead of grunt?

File: ui/packages/admin-federation-ui/src/main/resources/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ **/
+ -->
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+    <title>Federation</title>
+    <meta http-equiv="x-ua-compatible" content="IE=Edge">
+    <!-- Stylesheets -->
+    <link href="/webjars/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
```
Comment (brendan-hofmann) [inline ui/packages/admin-federation-ui/src/main/resources/index.html]: ❗️ If you want to be context-aware, the webjars need to be relative as well.

File: ui/packages/admin-federation-ui/src/main/webapp/main.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+/*jslint nomen:false, -W064 */
+
+(function () {
+    'use strict';
+
+    require.config({
+
+        paths: {
+
+            bootstrap: '/webjars/bootstrap/3.3.7/dist/js/bootstrap.min',
```
Comment (brendan-hofmann) [inline ui/packages/admin-federation-ui/src/main/webapp/main.js]: ❗️ These webjars are absolute as well.

**Comment by alexaabrd** [comment]
@brendan-hofmann I got rid of a lot of those urls but the ones that are left should all be relative now.  Also the build is using `ace bundle` instead of `grunt build` now 😄 

@shaundmorris I added a note about this endpoint in the section about managing the registries. I think that should suffice but can make changes if want more detail

File: distribution/docs/src/main/resources/content/_configuring/connecting-to-registry.adoc [hunk 3]
```diff
@@ -15,3 +15,8 @@ Use the registry to subscribe to and federate with other instances of ${branding
 ====
 The *Node Information* and *Remote Registries* tabs appear in both the ${ddf-registry} application and the ${ddf-catalog} application.
 ====
+
+[NOTE]
+====
+For direct federation configuration, sources and registries can be configured at https://{FQDN}:{PORT}/admin/federation.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_configuring/connecting-to-registry.adoc]: This seems to me. Not sure if @ricklarsen has any thoughts on it.

**Pull request merged**